docs/docusaurus/src/components/NavbarItems/GithubNavbarItem/index.js

```diff
@@ -1,22 +1,28 @@
 import React, {useEffect, useState} from 'react';
 import styles from './styles.module.scss';
 import useBaseUrl from "@docusaurus/useBaseUrl";
-export default function GithubNavbarItem({ owner, repository }) {
+
+export default function GithubNavbarItem({ owner, repository, className }) {
 
     const [starsCount, setStarsCount] = useState('0')
     const [forksCount, setForksCount] = useState('0')
     const [showGithubBadgeInfo, setShowGithubBadgeInfo] = useState(true)
+    const MOBILE_BREAKPOINT = 996;
 
     useEffect(() => {
-        fetch(`https://api.github.com/repos/${owner}/${repository}`)
-            .then(response => response.json())
-            .then(data => {
-                setStarsCount(formatCompactNumber(data.stargazers_count))
-                setForksCount(formatCompactNumber(data.forks_count))
-                setShowGithubBadgeInfo(true)
-            }).catch( _ => {
-                setShowGithubBadgeInfo(false)
-            })
+        if(window.innerWidth > MOBILE_BREAKPOINT){
+            fetch(`https://api.github.com/repos/${owner}/${repository}`)
+                .then(response => response.json())
+                .then(data => {
+                    setStarsCount(formatCompactNumber(data.stargazers_count))
+                    setForksCount(formatCompactNumber(data.forks_count))
+                    setShowGithubBadgeInfo(true)
+                }).catch( _ => {
+                    setShowGithubBadgeInfo(false)
+                })
+        } else {
+            setShowGithubBadgeInfo(false)
+        }
     }, []);
```

Comment:
this is not a response implementation, maybe try adding `window.innerWidth` in the dependency array.